platform/bazel/assert.bzl

```diff
@@ -1,11 +1,11 @@
 load("//bazel:util.bzl", "escape")
 
-def build_assert(name, src, message, condition, deps=None, visibility=None):
+def build_assert(name, src, message, condition, deps = None, visibility = None):
     cmdbase = "if ( {} ); then " + \
-                  "cp $(location {}) $@; " + \
+              "cp $(location {}) $@; " + \
```

Comment:
extremely pedantic nitpick: can we indent these blocks within the string literals